### ISSUE TITLE
Controls not working?

### ISSUE THREAD

**Issue by TrebuhD** [Codes4Fun/RBDOOM-3-BFG#26]
Are some of the the in-game controls not working for anyone else? I can navigate the main menu and accept with the right trigger, but it won't do anything with the in-game consoles. Only walking works fine.

**Comment by Codes4Fun** [comment]
Not all in-game consoles can be interacted with, even though they may show a mouse cursor on them. And keep in mind they work exactly the same as the main menu and PDA, you control a mouse cursor with your head and the right trigger to select things. You would need to be more specific on what is happening and maybe find a console that you know should work but doesn't.

**Comment by TrebuhD** [comment]
Thanks for the response. I can't use any of the in-game PDAs, including the one to call the elevator in the beginning of the game. I also can't talk to any of the NPCs, and can't turn the flashlight on. I tried reinstalling, it didn't help. AFAIK I'm the only one with this problem.

**Comment by Codes4Fun** [comment]
You can only turn the flashlight on after the first map just after the start of "mars underground", but you should have been able to talk to people.

The PDA is different then the in-game consoles, the PDA is activated with the left menu button but only after talking to the receptionist.

What mode are you playing in, standing or seated? (standing is default if you change nothing)

Also if you have motion controllers, make sure they are both on, if you seem them both in SteamVR then the game should also be able to use them.

Maybe also try using the Dev in the main menu and going to another level besides Mars City 1, like Mars City 2 for example, and you can try using the flashlight and guns there.

**Comment by TrebuhD** [comment]
I have motion controllers and they're definitely turned on :). I'm in Mars underground, got this far without touching or using anything except walking. I got my gun, but it's useless. If I switch to the desktop view in SteamVR and click on the Doom window using the Vive wand trigger, the gun shoots normally.

I haven't had similar issues with any other games. I've installed this mod over a fresh DOOM3 BFG install from Steam. 

Steps I've taken to resolve this issue:

- Deleted the _Users/$user/Saved Games/id Software/Doom 3 BFG VR_ folder
- Reinstalled the game & mod

Any other files or registry keys I could try deleting before reinstalling again? I appreciate the help. At this point I'm thinking of reinstalling Windows.

**Comment by Codes4Fun** [comment]
> Deleted the Users/$user/Saved Games/id Software/Doom 3 BFG VR folder

The actual folder to delete should start with "RB": "%userprofile%\Saved Games\id Software\RBDOOM 3 BFG"

The folder you have deleted is for the [other VR mod by KozGit](https://github.com/KozGit/DOOM3-BFG-RIFT/releases).

Make sure you are installing and running RBDoom3BFG.exe from my [release page](https://github.com/Codes4Fun/RBDOOM-3-BFG/releases).

Do you have a regular game controller? If you do, try and see if it will work, you might need to enable it in the Settings. The way I have it setup is that some Vive buttons are treated as joystick buttons, so if a regular gamepad doesn't work then possibly it is a different issue.

**Comment by vovcacik** [comment]
I can confirm this issue. Not sure how to reproduce it, but it happened to my brother too (and he got completely separate setup).

However the fix is rather easy, just use "Dev" menu option to jump to mission *Mars 2* (or something like that). It should be all fine in that mission **and then you can go back to the first mission and it will all work fine.**
